app/models/concerns/authorization_concern.rb

```diff
@@ -40,6 +40,7 @@ module AuthorizationConcern
     # field :confirmation_sent_at, type: Time
     # field :unconfirmed_email,    type: String # Only if using reconfirmable
 
+    validates :password, format: { without: /\s/, message: "Password must not contain spaces" }
```

Comment:
Unblocking comment -

How about including this in `password_complexity` def itself. And, this method needed a refactor, it's doing same checks multiple times.

Reply:
We will be using a separate ticket for code refactor.